docs/setup_vscode.md

```diff
@@ -155,7 +158,32 @@ $ code .
 ```
 </div>
 
-<img src="images/vscode020.png" width="768px" />
+<img src="images/vscode020.png" width="512px" />
+
+### Windows  
```

Comment:
This information might also be needed by Visual Studio users.  Would it make sense to put the detailed description with the screenshot in `setup_wsl.md` and then link to it here with a pro-tip?

Reply:
Seems like a good idea, I'll add it in